### PR TITLE
MULE-15904: Allow to discover mule services from a particular folder …

### PR DESCRIPTION
**Pull request by gsfernandes**
…location.

### CODE REVIEW

File: modules/service/src/main/java/org/mule/runtime/module/service/internal/discoverer/FileSystemServiceProviderDiscoverer.java [hunk 1]
```diff
@@ -77,7 +98,7 @@ public FileSystemServiceProviderDiscoverer(ArtifactClassLoader containerClassLoa
   private List<ServiceDescriptor> getServiceDescriptors(ServiceDescriptorFactory serviceDescriptorFactory)
       throws ServiceResolutionError {
 
-    final File[] serviceDirectories = getServicesFolder().listFiles(File::isDirectory);
+    final File[] serviceDirectories = this.servicesFolder.listFiles(File::isDirectory);
```
Comment (elrodro83) [inline modules/service/src/main/java/org/mule/runtime/module/service/internal/discoverer/FileSystemServiceProviderDiscoverer.java]: why this change?

**Comment by gsfernandes** [comment]
@elrodro83 changed the field to avoid confusions.
